src/types.ts

```diff
@@ -3,6 +3,26 @@ import { stardust } from '@nebula.js/stardust';
 
 export type Align = 'left' | 'center' | 'right';
 
+type Size = {
+  w: number;
+  h: number;
+};
+export interface ViewService {
+  qLeft: number;
+  qTop: number;
+  qWidth: number;
+  qHeight: number;
+  scrollLeft: number;
+}
+export interface SnapshotData {
+  content?: {
+    scrollLeft: number;
+  };
+  object: {
```

Comment:
when object.size is assigned to snapshotData?

Reply:
Good point! 
It is from sense client https://github.com/qlik-trial/sense-client/blob/master/web/assets/objects/snapshot/snapshot-helper.js#L81 but we don't need it in sn-table.